_includes/layouts/permalink.html

```diff
@@ -0,0 +1,16 @@
+---
+layout: default
+---
+
+<div class="u-containProse u-md-textGrow1">
+  {% include resource-item.html, 
```

Comment:
These files are all using the `.html` extension, but I notice these `{%` tags. What template engine is being used? Is there a reason we aren't using its file extension so syntax highlighting will be easier?

Reply:
Eleventy's default template language for HTML is liquid, so that's what's being used here. These are the same extensions leftover from the Jekyll version, so no strategic choice was made (by me) to remove them. I don't think Eleventy will care if we add them, but it will add a good bit of noise to the diff for this PR. Maybe easier to do that in a separate PR?

Reply:
https://github.com/cloudfour/pwastats/issues/210